platforms/android/src/org/crosswalk/engine/XWalkCordovaUiClient.java

```diff
@@ -172,12 +175,15 @@ public void onPageLoadStopped(XWalkView view, String url, LoadStatus status) {
     // File Chooser
     @Override
     public void openFileChooser(XWalkView view, final ValueCallback<Uri> uploadFile, String acceptType, String capture) {
-        uploadFile.onReceiveValue(null);
+        if (mFileChooser == null) {
+            mFileChooser = new XWalkFileChooser(parentEngine.cordova.getActivity());
```

Comment:
http://img.blog.csdn.net/20150104183450879
Inflation will be done with the default theme for the system on which you are running, not what’s defined in your application.
Did it verified it worked finely with Application Context?